### PR TITLE
Audited examples in endpoint.r, see TODO items for further edits needed.

### PR DESCRIPTION
**Pull request by tinatownes**
### Checklist

Please:

- [x] Make sure this PR is against "dev", not "main" (unless this is a release
      PR).
- [x] Request a review from one of the current epidatr main reviewers:
      brookslogan, dshemetov, nmdefries, dsweber2.
- [ ] Makes sure to bump the version number in `DESCRIPTION`. Always increment
      the patch version number (the third number), unless you are making a
      release PR from dev to main, in which case increment the minor version
      number (the second number).
- [ ] Describe changes made in NEWS.md, making sure breaking changes
      (backwards-incompatible changes to the documented interface) are noted.
      Collect the changes under the next release number (e.g. if you are on
      1.7.2, then write your changes under the 1.8 heading).

### Magic GitHub syntax to mark associated Issue(s) as resolved when this is merged into the default branch

- Resolves #{issue number}

### CODE REVIEW

**Comment by nmdefries** [comment]
@dshemetov Can you take a look at this?

File: README.md
Comment (dshemetov) [inline README.md:19]: This link says it's been moved to: https://archive.cdc.gov/

```suggestion
(CDC)](https://archive.cdc.gov/), private partners
```

File: README.md
Comment (dshemetov) [inline README.md:85]: I think the original is closer to correct because it's considered canonical by CRAN (I don't fully understand _why_ they want things that way, but [this is an example](https://stackoverflow.com/questions/64257455/cran-url-not-in-canonical-form) of the issue CRAN would raise). So, if anything the URL should be 
```suggestion
[here](https://CRAN.R-project.org/package=epidatr).
```

File: R/endpoints.R
Comment (dshemetov) [inline R/endpoints.R:167]: This works for me thank you!

File: R/endpoints.R
Comment (dshemetov) [inline R/endpoints.R:1851]: This one too, thank you!

**Review by dshemetov — APPROVED** [review]
Going to approve this so you can just merge after fixing, but please make sure to address the two minor issues (both URLs) I raised before merging. The CRAN url in particular will make the next release harder if not addressed.

**Pull request merged**
